versioned_docs/version-24.10/releases/centreon-os.mdx

```diff
@@ -182,6 +182,32 @@ Release date: `October 31, 2024`
 
 ## Centreon Collect
 
+### 24.10.3
+
+Release date: `December 18, 2024`
+
+<details open>
+  <summary>Enhancements</summary>
+
+- [CMA] Added a native CPU check for Windows & Linux.
+- [CMA] Added a native Memory check for Windows.
+- [CMA] Added a native Uptime check for Windows.
+- [CMA] Added a native Storage check for Windows.
+- [CMA] Agent checks scheduling has been optimized to avoid delay in case of checks timeouts.
+- [CMA] Centreon Monitoring Agent & installer are now signed.
```

Comment:
```suggestion
- [CMA] Added a native CPU check for Windows & Linux.
- [CMA] Added native checks for Windows: Memory, Uptime and Storage.
- [CMA] Agent check scheduling has been optimized to avoid delay in case of check timeouts.
- [CMA] The Centreon Monitoring Agent and its installer are now signed.
```